sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java

```diff
@@ -353,11 +357,12 @@ public O execute(Client client, Duration timeout) throws TimeoutException, Prech
                 throw new MaxAttemptsExceededException(lastException);
             }
 
-            if (Instant.now().isAfter(timeoutTime)) {
+            Duration deadline = Duration.between(Instant.now(), timeoutTime);
```

Comment:
It's a bit misleading to call this deadline, since the actual deadline will be under 10s. This is more a timeout for all requests whereas deadline is per request. So recommend renaming to `timeout`.

Reply:
I renamed it to 'currentTimeout' to differentiate from the method parameter which is already called timeout.